src/wallet/crypter.h

```diff
@@ -116,6 +116,7 @@ class CCryptoKeyStore : public CBasicKeyStore
 {
 private:
     CryptedKeyMap mapCryptedKeys;
+    KeyMap mapBip47Keys;
```

Comment:
I don't think this separate `KeyMap` is needed. you should find some way of using `mapCryptedKey` for what you need, from what I can see in the code there isn't a reason why we need another object.